robot/basestation/templates/elements/navigation/NavigationData.html

```diff
@@ -0,0 +1,109 @@
+                                    <li><button id="antenna-long-DMS-btn" class="btn btn-primary btn-sm">Degrees Minutes Seconds</button></li>
+                                </ul>
+                            </div>
+                        </fieldset>
+                        <hr>
+                        <fieldset id="antenna-bearing-fieldset">
+                            <div class="input-group" id="antenna-bearing-input-group">
+                                <span class="input-group-text">Bearing</span>
+                                <input id="antenna-bearing-input" type="text" class="form-control antenna-input-field" placeholder="Enter">
+                                <button id="antenna-bearing-change-btn" class="btn btn-warning antenna-change-btn">Change</button>
+                            </div>
+                        </fieldset>
+                    </div>
+                    <div class="modal-footer">
+                        <button id="antenna-confirm-btn" type="button" class="btn btn-success">Confirm</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <br>
+        <br>
+        <div id="goal-stats-list">
+            <h4 align="center">Goals</h4>
+
+            <ul class="list-group" id="goal-stats-list">
+                <div class="row px-3">
+                    <li class="list-group-item col-md-6"><label for="goal-stats-lat">Latitude</label><span id="goal-stats-lat" class="float-right">----</span></li>
+                    <li class="list-group-item col-md-6"><label for="goal-stats-long">Longitude</label><span id="goal-stats-long" class="float-right">----</span></li>
+                </div>
+                <li class="list-group-item"><label for="goal-stats-distance">Distance to goal</label><span id="goal-stats-distance" class="float-right">----</span></li>
+                <li class="list-group-item"><label for="goal-stats-rover-heading">Recommended Rover Heading</label><span id="goal-stats-rover-heading" class="float-right">----</span></li>
+            </ul>
+        </div>
+
+        <button type="button" id="toggle-goal-modal-btn" class="btn btn-primary" data-toggle="modal" data-target="#goal-modal">Add Goal</button>
+
+        <div class="modal fade" id="goal-modal">
+            <div class="modal-dialog modal-lg">
+                <div class="modal-content" id="goal-modal-content">
+                    <div class="modal-header" id="goal-modal-header">
+                        <button id="new-goal-coordinates-btn" class="btn btn-primary">Add new coordinates</button>
+                    </div>
+                    <div class="modal-body" id="goal-modal-body">
+                        <div id="goal-modal-body-content">
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
```

Comment:
1. The whole file should be un-indented
2. Is this an extra rogue `</div>` ? If so please remove it

Reply:
Fixed